bbb-common-web/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
```

Comment:
I am for improving this and running it under a dedicated subnet and forcing it IP address. Any additional piece of advice from the community is welcome.